about us/about_us.js

```diff
@@ -0,0 +1,341 @@
+window.onload = () => {
+  renderCandidateTraits(candidateTraits);
+  renderProgramStageInfo(programStages);
+  renderKnownObstacles(knownObstacles);
+};
+
+const candidateTraitsBox = document.getElementById("candidate-traits");
+candidateTraitsBox.classList.add("candidate-traits");
```

Comment:
if you are adding this class globally, why not just put it on the element in html?